ui/app/routes/policies/policy.js

```diff
@@ -2,15 +2,23 @@ import Route from '@ember/routing/route';
 import withForbiddenState from 'nomad-ui/mixins/with-forbidden-state';
 import WithModelErrorHandling from 'nomad-ui/mixins/with-model-error-handling';
 import { inject as service } from '@ember/service';
+import { hash } from 'rsvp';
 
 export default class PoliciesPolicyRoute extends Route.extend(
   withForbiddenState,
   WithModelErrorHandling
 ) {
   @service store;
-  model(params) {
-    return this.store.findRecord('policy', decodeURIComponent(params.name), {
-      reload: true,
+  async model(params) {
+    return await hash({
+      policy: this.store.findRecord('policy', decodeURIComponent(params.name), {
+        reload: true,
+      }),
+      tokens: this.store
+        .peekAll('token')
+        .filter((token) =>
+          token.policyNames?.includes(decodeURIComponent(params.name))
+        ),
     });
   }
```

Comment:
I'm not opposed to using Promise.all, but it seems less readable and idiomatic for our models to have otherwise-unused consts that are destructured later. Are there good reasons to avoid rsvp.hash?